packages/iTwinUI-react/src/core/Table/columns/actionColumn.tsx

```diff
@@ -49,6 +49,16 @@ export const ActionColumn = <T extends Record<string, unknown>>({
     disableReordering: true,
     Header: ({ allColumns, dispatch, state }: HeaderProps<T>) => {
       const [isOpen, setIsOpen] = React.useState(false);
+
+      const buttonRef = React.useRef<HTMLButtonElement>(null);
+      const hasToggledVisibility = React.useRef(false);
+      React.useEffect(() => {
+        if (hasToggledVisibility.current) {
```

Comment:
Kind of a hacky way but I tried to add `scrollIntoView` inside `onClick` but it didn't work because I guess DOM wasn't still changed, also tried to set body scroll inside `Table.tsx` in several places but also didn't work because of the same - DOM wasn't changed yet.

Reply:
Does `queueMicrotask` help?

Reply:
`setTimeout` did not help, so I guess this one neither.

Reply:
Hm, ok, I needed to refresh the page for my changes to work. So `setTimeout` works but there is visible scroll and `queueMicrotask` also worked, so gonna change to that.